tests/helpers.py

```diff
@@ -25,7 +25,6 @@ def setUp(self):
 
     db_name = "faunadb-python-test"
     self.db_ref = Ref("databases", db_name)
-    # TODO: See `core` issue #1975
```

Comment:
This issue is still open. When it's fixed, I'd like to check for existence rather than catching errors.